### ISSUE TITLE
The Accuracy of Model Training

### ISSUE THREAD

**Issue by AngLancer** [Shuijing725/CrowdNav_DSRNN#17]
Hello, I'm sorry to interrupt you.  I used the parameters in `/data/example_model/configs/config.py` to train on my machine and got `/data/dummy/checkpoints/27776.pt`, and I used `test.py` to test my training results, the results are: TEST has success rate: 0.21, collision rate: 0.12, timeout rate: 0.67, NAV time: 20.78, total reward: 9.6464.
This is far from the 90% correct rate in your `/data/example_model/test/test_27776.pt.log`. Why is this?
My GPU is NVIDIA Tesla K80 and my CPU is three Xeon E5-2678 V3.
Python 3.7, CUDA 10.2, cuDNN 7.6, Pytorch 1.6.0, Ubuntu 18.04

**Comment by Shuijing725** [comment]
Can you provde the training curve and the checkpoint you tested with?

**Comment by AngLancer** [comment]
Sorry, I didn't save the previous results. I ran again with 2080ti, and the success rate became 90%, consistent with yours. But I still don't know why the success rate of K80 is so low.
Excuse me again, I see that the success rate in `/data/example_model/test/test_27776.pt.log` is 90%, and the success rate of Fov360 in Figure 5 in the paper is 98%. Under what circumstances has such a high accuracy been achieved? Thank you.

**Comment by Shuijing725** [comment]
Your new results look normal. If you change the random seed, change the checkpoint, or finetune the hyperparameters, you may get better results.
